README.md

```diff
@@ -21,6 +21,17 @@ $ npm install --save resin-register-device
 Documentation
 -------------
 
+Resin-Register-Device exports a factory function, which must be called with a dependencies object, containing a configured [`resin-request`](https://github.com/resin-io-modules/resin-request) instance.
+
+Example:
+```coffee
+deviceRegister = require('resin-register-device')({
+	request: require('resin-request')({
+		dataDirectory: '/opt/cache/resin'
```

Comment:
What happens with this directory if we never use tokens? For this module in particular every request is made with an api key and not a token - I'm wondering for how it'll interact with the supervisor